### PR TITLE
Query support

### LINKED ISSUE #25: Add query support
Input: text field to input JSON
Output: query response
API: `CosmWasmClient.queryContractSmart`

### PR DESCRIPTION
**Pull request by abefernan**
Closes #25

### CODE REVIEW

File: package.json
Comment (webmaster128) [inline package.json:35]: This is unused, right? I think you can just remove the "Adds new dependencies" commit from this PR

File: src/pages/contract/QueryContract.tsx
Comment (webmaster128) [inline src/pages/contract/QueryContract.tsx:74]:
```suggestion
              <pre className="mb-0">{JSON.stringify(queryObject.result, null, "  ")}</pre>
```

No need for extra spacing I think. list-group-item has a padding already

File: src/pages/contract/QueryContract.tsx
Comment (webmaster128) [inline src/pages/contract/QueryContract.tsx:85]:
```suggestion
            <span title="The query response">Response:</span>
```

**Review by webmaster128 — COMMENTED** [review]
Looking good.

I'm not sure if we really need the formatted input, but I don't have a better idea right now. Ideally we find a multi-line JSON input component with highlighting an in-line errors. Then one field would be enought.

**Comment by abefernan** [comment]
> Ideally we find a multi-line JSON input component with highlighting an in-line errors. Then one field would be enought.

I found this, should I integrate it in this PR?: https://www.npmjs.com/package/react-json-editor-ajrm

**Comment by webmaster128** [comment]
> I found this, should I integrate it in this PR?: https://www.npmjs.com/package/react-json-editor-ajrm

Cool! Yeah, let's give it a try

**Review by webmaster128 — APPROVED** [review]

**Review by webmaster128 — COMMENTED** [review]
Waiting for the new input component before merge

File: src/pages/contract/QueryContract.tsx
Comment (webmaster128) [inline src/pages/contract/QueryContract.tsx:58]: Can we use this as a default?

```json
{
  "get_balance": {
    "address": "cosmos1zk4hr47hlch274x28j32dgnhuyewqjrwxn4mvm"
  }
}
```

File: src/pages/contract/QueryContract.tsx
Comment (webmaster128) [inline src/pages/contract/QueryContract.tsx:55]: Can we configure this component to only accept JSON, not all JavaScript?

Comment (webmaster128) [reply src/pages/contract/QueryContract.tsx:55]: Answer seems to be no. See e.g. https://github.com/AndrewRedican/react-json-editor-ajrm/issues/120

**Pull request merged**
